lib/cc/analyzer/issue_validations/severity_validation.rb

```diff
@@ -0,0 +1,40 @@
+module CC
+  module Analyzer
+    module IssueValidations
+      class SeverityValidation < Validation
+        INFO  = "info".freeze
+        MINOR = "minor".freeze
+        MAJOR = "major".freeze
+        CRITICAL = "critical".freeze
+        BLOCKER = "blocker".freeze
+
+        DEPRECATED_SEVERITIES = [
+          NORMAL = "normal".freeze
+        ].freeze
+
+        VALID_SEVERITIES = ([
+          INFO,
+          MINOR,
+          MAJOR,
+          CRITICAL,
+          BLOCKER,
+        ] + DEPRECATED_SEVERITIES).freeze
+
+        def valid?
+          severity.nil? || VALID_SEVERITIES.include?(severity)
```

Comment:
I considered checking for `.present?` here, but I don't think we need or want that. The overwrite to default `'minor` only checks for `nil` [here](https://github.com/codeclimate/codeclimate/commit/0938c70ab78b9075966bee3ae006265dff8aa5fc#diff-e013abc9492360e8274e92f76b9c6071R52). Open to other ideas though.